platform.txt

```diff
@@ -83,8 +83,8 @@ compiler.elf2hex.extra_flags=
 
 ## generate file with git version number
 ## needs bash, git, and echo
-recipe.hooks.core.prebuild.1.pattern="{runtime.tools.python.path}/python" "{runtime.tools.signing}" --mode header --publickey "{build.source.path}/public.key" --out "{build.path}/core/Updater_Signing.h"
-recipe.hooks.core.prebuild.2.pattern="{runtime.tools.python.path}/python" "{runtime.tools.makecorever}" --build_path "{build.path}" --platform_path "{runtime.platform.path}" --version "unix-{version}"
+recipe.hooks.sketch.prebuild.1.pattern="{runtime.tools.python.path}/python" "{runtime.tools.signing}" --mode header --publickey "{build.source.path}/public.key" --out "{build.path}/core/Updater_Signing.h"
+recipe.hooks.core.prebuild.1.pattern="{runtime.tools.python.path}/python" "{runtime.tools.makecorever}" --build_path "{build.path}" --platform_path "{runtime.platform.path}" --version "unix-{version}"
```

Comment:
This breaks the packager script.  For build-boards-manager, we remove this line by name since the version will be pre-gen's in the Arduino tarball.

Reply:
Same comment about the .1. not being needed, as above.

Reply:
Both fixed. PR branch is updated.

Could I have easily spotted this myself, e.g. by running some test scripts?

Reply:
Not easily, unfortunately.  The `package/build_boards_manager.sh` script is run on releases to generate the uploaded tarball.  While the script w/o any changes would still produce a *legal* platform.txt, it would try and regenerate the GIT version and fail whenever anything was built.

You'd have to run the script, make a json, upload it and your tarball to a private webserver, add that JSON url to your boards manager in the IDE, then install it (after removing/hiding your git copy).